openedx_events/event_bus/avro/tests/schemas/org+openedx+learning+program+certificate+awarded+v1_schema.avsc

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "CloudEvent",
+  "type": "record",
+  "doc": "Avro Event Format for CloudEvents created with openedx_events/schema",
+  "fields": [
+    {
+      "name": "program_certificate",
+      "type": {
+        "name": "ProgramCertificateData",
+        "type": "record",
+        "fields": [
+          {
+            "name": "user",
+            "type": {
+              "name": "UserData",
+              "type": "record",
+              "fields": [
+                {
+                  "name": "id",
+                  "type": "long"
+                },
+                {
+                  "name": "is_active",
+                  "type": "boolean"
+                },
+                {
+                  "name": "pii",
+                  "type": {
+                    "name": "UserPersonalData",
+                    "type": "record",
+                    "fields": [
+                      {
+                        "name": "username",
+                        "type": "string"
+                      },
+                      {
+                        "name": "email",
+                        "type": "string"
+                      },
+                      {
+                        "name": "name",
+                        "type": "string"
+                      }
+                    ]
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "name": "program",
+            "type": {
+              "name": "ProgramData",
+              "type": "record",
+              "fields": [
+                {
+                  "name": "uuid",
+                  "type": "string"
+                },
+                {
+                  "name": "title",
+                  "type": "string"
+                },
+                {
+                  "name": "program_type",
+                  "type": "string"
+                }
+              ]
+            }
+          },
+          {
+            "name": "uuid",
+            "type": "string"
+          },
+          {
+            "name": "status",
+            "type": "string"
+          },
+          {
+            "name": "url",
+            "type": "string"
+          },
+          {
+            "name": "certificate_available_date",
+            "type": [
+              "null",
+              "string"
+            ],
+            "default": null
```

Comment:
what does this default do?

Reply:
I realized I read this comment wrong. My brain interpreted it as "what does this default TO" which is definitely not what it says. 

Actually answering your question, I'll take the explanation from the Avro spec doc:
```
default: A default value for this field, used when reading instances that lack this field (optional). Permitted values depend on the field's schema type, according to the table below. Default values for union fields correspond to the first schema in the union. Default values for bytes and fixed fields are JSON strings, where Unicode code points 0-255 are mapped to unsigned 8-bit byte values 0-255.
```